pages/panel.js

```diff
@@ -38,5 +61,13 @@ PanelPage.propTypes = {
 };
 
 export default withRedux(
-  store
+  store,
+  state => ({
+    indicators: state.indicators
+  }),
+  dispatch => ({
+    getIndicators() {
```

Comment:
This is the same as: 
`getIndicators() {
    dispatch(getIndicators());
}`=
`getIndicators,`
So... this is not a blocker. I only want to let you know that we use the function only if we want to dispatch 2 actions at the same time. For example, if you want to update the url